bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/util/ThingHandlerHelper.java

```diff
@@ -15,20 +15,30 @@
  * This class provides utility methods related to the {@link ThingHandler} class.
  *
  * @author Markus Rathgeb - Initial contribution
+ * @author Simon Kaufmann - added UNKNOWN
  */
 public class ThingHandlerHelper {
 
     private ThingHandlerHelper() {
     }
 
     /**
+     * Checks if the given state indicates that a thing handler has been initialized.
+     *
+     * @return true if the thing handler has been initialized, otherwise false.
+     */
+    public static boolean isHandlerInitialized(final ThingStatus thingStatus) {
```

Comment:
> Or do you think it needs to much load?

Yes, this is really just filtering events and I wouldn't want this place to require a lookup in the registry.